docs/en/sql-reference/sql-statements/Data Manipulation/ROUTINE LOAD.md

```diff
@@ -163,10 +163,18 @@ FROM data_source
 
         Whether to enable strict mode, the default is on. If turned on, the column type transformation of non-null raw data is filtered if the result is NULL. Specified as "strict_mode" = "true"
             
-    5. timezone
+    5. `timezone`
 
         Specifies the time zone in which the job will be loaded. The default by using session variable's timezone. This parameter affects all function results related to the time zone involved in the load.
 
+    6. `format`
+
+        Specifies the format of the imported data, which is CSV by default and supports json format.
+
+    7. `jsonpath`、`jsonpath_file`
```

Comment:
jsonpath_file is not supported any more, please remove it from document

lack string_outer_array explain